wrapper/desktop/CMakeLists.txt

```diff
@@ -31,4 +31,6 @@ INCLUDE("${CMAKE_CURRENT_LIST_DIR}/../../CMakeLists.txt")
 
 ADD_EXECUTABLE(${OPENMAPPER_APP} ${OM_APP_RESOURCES})
 
-TARGET_LINK_LIBRARIES(${OPENMAPPER_APP} debug ${OPENMAPPER_LIB} optimized ${OPENMAPPER_LIB})
+TARGET_LINK_LIBRARIES(${OPENMAPPER_APP}
+  debug ${OPENGL_LIBRARIES} optimized ${OPENGL_LIBRARIES}
```

Comment:
yes ive been thinking about it too. if the desktop layer remains quite thin, we could also keep it entirely here instead of a separate repo...